mne/preprocessing/eog.py

```diff
@@ -49,15 +49,19 @@ def find_eog_events(raw, event_id=998, l_freq=1, h_freq=10,
     eog_inds = _get_eog_channel_index(ch_name, raw)
     logger.info('EOG channel index for this subject is: %s' % eog_inds)
 
-    eog, _ = raw[eog_inds, :]
+    # Reject bad segments.
+    eog, times = raw.get_data(picks=eog_inds, reject_by_annotation='omit',
```

Comment:
Just noticed that the ``reject_by_annotation`` in ``create_eog_epochs`` will function a bit differently. It will reject the epochs that overlap with the bad segments whereas this will find the events from the good data. Should this be unified or is it okay to have them work differently?